tol_lab_share/messages/output_traction_message.py

```diff
@@ -216,6 +217,7 @@ def __init__(self):
         super().__init__(Input(self))
         self._requests: Dict[int, OutputTractionMessageRequest] = {}
         self._sent = False
+        self._validate_certificates = get_config("").CERTIFICATES_VALIDATION_ENABLED
```

Comment:
just checking get_config with empty string is correct? gets defaults file?

Reply:
yes, now reading it does not seem so nice, because "" is the default in any case. I want to use the defaults for get_config (that will use the environment defined settings file, or config.defaults if nothing defined); but that passing of "" is very confusing and not needed. I'll change it